.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All: what do we think about checking in IDE files? I'm using Pycharm, but I've refrained from checking in the `.idea/` directories because I assume other people don't want them there.